datafusion/physical-plan/src/sorts/sort.rs

```diff
@@ -765,7 +766,12 @@ impl DisplayAs for SortExec {
                 let expr: Vec<String> = self.expr.iter().map(|e| e.to_string()).collect();
                 match self.fetch {
                     Some(fetch) => {
-                        write!(f, "SortExec: fetch={fetch}, expr=[{}]", expr.join(","))
+                        write!(
+                            f,
+                            // TODO should this say topk?
```

Comment:
I'm not sure if we would like to do this? I think there are some other `ExecutionPlan` nodes that have the algorithm depend on one of the parameters (for example: HashAggregate `mode`s) .

Reply:
I think in general it would be good to be able to tell what operator was going to be used from looking at the plan. However, I think we can do so as a follow on PR -- I can file a ticket.